website/docs/d/iam_access_group_template.html.markdown

```diff
@@ -0,0 +1,125 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_access_group_template"
+description: |-
+  Get information about iam_access_group_template
+subcategory: "IAM Access Groups"
```

Comment:
Already we have subgroup  Identity & Access Management (IAM) for other access group res/ds can we use the same

Reply:
Updated